addons/hr_recruitment/static/src/views/form_view.js

```diff
@@ -4,9 +4,18 @@ import { registry } from '@web/core/registry';
 
 import { formView } from '@web/views/form/form_view';
 import { FormController } from '@web/views/form/form_controller';
-import { session } from '@web/session';
-
+import { onWillStart } from "@odoo/owl";
+import { useService } from '@web/core/utils/hooks';
 export class InterviewerFormController extends FormController {
+    setup() {
+        super.setup();
+        this.user = useService("user");
+        onWillStart(async () => {
+            this.isAdmin = await this.user.hasGroup("hr_recruitment.group_hr_recruitment_manager");
+            this.isInterviewer = await this.user.hasGroup("hr_recruitment.group_hr_recruitment_interviewer");
+            this.isUser = await this.user.hasGroup("hr_recruitment.group_hr_recruitment_user");
```

Comment:
I would move these checks in the python part, so that the js class is accessed only for user with `group_hr_recruitment_interviewer` but without `group_hr_recruitment_user` or `group_hr_recruitment_manager`